src/components/PersonaCard/PersonaCard.Square.html

```diff
@@ -15,31 +15,39 @@
     </div>
   </div>
   <ul class="ms-PersonaCard-actions">
-    <li id="chat" class="ms-PersonaCard-action is-active"><i class="ms-Icon ms-Icon--chat"></i></li>
-    <li id="phone" class="ms-PersonaCard-action"><i class="ms-Icon ms-Icon--phone"></i></li>
-    <li id="video" class="ms-PersonaCard-action"><i class="ms-Icon ms-Icon--video"></i></li>
-    <li id="mail" class="ms-PersonaCard-action"><i class="ms-Icon ms-Icon--mail"></i></li>
-    <li class="ms-PersonaCard-overflow" alt="View profile in Delve" title="View profile in Delve">View profile</li>
-    <li id="org" class="ms-PersonaCard-action ms-PersonaCard-orgChart"><i class="ms-Icon ms-Icon--org"></i></li>
+    <li id="phoneAction" class="ms-PersonaCard-action" data-detailsTargetId="detail-2b"><i class="ms-Icon ms-Icon--phone"></i></li>
+    <li id="chatAction" class="ms-PersonaCard-action" data-detailsTargetId="detail-1b"><i class="ms-Icon ms-Icon--chat"></i></li>
+    <li id="videoAction" class="ms-PersonaCard-action" data-detailsTargetId="detail-3b"><i class="ms-Icon ms-Icon--video"></i></li>
+    <li id="mailAction" class="ms-PersonaCard-action is-active" data-detailsTargetId="detail-4b"><i class="ms-Icon ms-Icon--mail"></i></li>
+    <li id="profileAction" class="ms-PersonaCard-overflow" data-detailsTargetId="detail-5b" data-actionUrl="http://dev.office.com/fabric" alt="View profile in Delve" title="View profile in Delve">View profile</li>
+    <li id="orgAction" class="ms-PersonaCard-action ms-PersonaCard-orgChart" data-detailsTargetId="detail-6b"><i class="ms-Icon ms-Icon--org"></i></li>
   </ul>
   <div class="ms-PersonaCard-actionDetailBox">
-    <ul id="detailList" class="ms-PersonaCard-detailChat">
-      <li id="chat" class="ms-PersonaCard-actionDetails detail-1">
+    <ul id="detail-1b" class="ms-PersonaCard-details">
+      <li  class="ms-PersonaCard-actionDetails">
         <div class="ms-PersonaCard-detailLine"><span class="ms-PersonaCard-detailLabel">Lync:</span> <a class="ms-Link" href="#">Start Lync call</a></div>
       </li>
-      <li id="phone" class="ms-PersonaCard-actionDetails detail-2">
+   </ul>
+   <ul id="detail-2b" class="ms-PersonaCard-details">
```

Comment:
Rather than connecting a data attribute (`data-detailsTargetId="detail-2b"`) to an ID, could we connect two data attributes? It would be best to avoid IDs entirely as they can easily be given names that unknowingly clash with existing styles. Better to stay clear of CSS Land entirely and use data attributes.